### ISSUE TITLE
[Suggestion] Per-datapack EMC Values

### ISSUE THREAD

**Issue by CraftSpider** [sinkillerj/ProjectE#2284]
**Tickets that do not conform to this template will be closed without comment**

Exact ProjectE version (do not say "latest", "latest on Curse", or similar): 

1.0.1B

Exact Forge version:

43.2.8

Desired Feature:

Instead of one massive `custom_emc.json`, I'd like to be able to split it up for re-use - I'd like to be able to make a datapack for 'Biome's You Go items' and another for 'Thermal items', so that I can re-use these files for packs with the same mods, rather than have to hunt through them to copy over just the bits I want. I think this is useful in part because most of the time, packs use the default EMC values, and these packs can be tuned to match that default balance, and be drop-in for anyone who wants the support.

**Comment by pupnewfster** [comment]
This is already possible via custom conversions which ProjectE uses to define defaults for things like vanilla.

Example file: https://github.com/sinkillerj/ProjectE/blob/mc1.19.x/example_custom_conversion.json (might be slightly out of date I forget)
ProjectE's own datapack that has conversions and values defined: https://github.com/sinkillerj/ProjectE/tree/mc1.19.x/src/datagen/generated/data/projecte/pe_custom_conversions

Basically any conversion json files found in a `pe_custom_conversions` subfolder of a datapack will be loaded.

**Comment by CraftSpider** [comment]
Ah, thank you, that's exactly the kind of thing I was looking for, and supports some stuff I was missing - the whole 'custom conversion' stuff looks great for not needing to manually propagate values across non-standard crafting methods. Thank you, sorry I missed it!
